include/internal/cryptlib.h

```diff
@@ -27,6 +27,7 @@
 
 #ifdef NDEBUG
 # define ossl_assert(x) ((x) != 0)
+# define ossl_is_null(x) ((x) == NULL)
```

Comment:
Interesting: clang is overanxious and fails because of the extra parentheses
```
ssl/packet.c:137:16: error: equality comparison with extraneous parentheses [-Werror,-Wparentheses-equality]
    if (((buf) == ((void*)0)))
```
https://travis-ci.org/openssl/openssl/jobs/428394151